src/compute/src/extensions/arrange.rs

```diff
@@ -247,7 +247,12 @@ where
     L: FnMut(&Tr) -> (usize, usize, usize) + 'static,
 {
     let scope = arranged.stream.scope();
-    let Some(logger) = scope.log_register().get::<ComputeEvent>("materialize/compute") else {return arranged};
+    let Some(logger) = scope
+        .log_register()
+        .get::<ComputeEvent>("materialize/compute")
+    else {
+        return arranged;
+    };
```

Comment:
Are the reformattings intended? Why do they occur?

Reply:
in general, rustfmt output is not stable between versions. I think it only gained the ability to format let/else statements quite recently, so it was probably just ignoring any let/else statement before.

Reply:
yeah, its not stable, but it should be backwards compatible with 1.71 like this!